TombEngine/Game/savegame.cpp

```diff
@@ -1151,7 +1151,7 @@ const std::vector<byte> SaveGame::Build()
 		particleInfo.add_b(particle->b);
 		particleInfo.add_col_fade_speed(particle->colFadeSpeed);
 		particleInfo.add_d_b(particle->dB);
-		particleInfo.add_sprite_index(particle->spriteIndex);
+		particleInfo.add_sprite_index(particle->SpriteSeqID); // TODO: Save SpriteID.
```

Comment:
It shouldn't be a todo, it should be added in this PR.
Additionally, `targetPos` field also should be saved in a savegame, otherwise your particle will fall through the floor and won't be disabled in time.